styles.css

```diff
@@ -3,65 +3,59 @@
   font-family: 'Poppins', sans-serif;
 }
 
-.head {
-  flex-direction: row-reverse;
-  align-items: center;
-  background-image: url(img/image_geometry_1.png);
-  display: flex;
-  justify-content: space-between;
-  background-repeat: no-repeat;
-  background-size: contain;
-  height: 70px;
+body {
+  background-color: #f9f9fa;
 }
```

Comment:
- [ ] Kindly remove the horizontal overflow in order to enhance the user experience. 😃 

## Your Design

![b11](https://user-images.githubusercontent.com/103386183/199991780-89a80649-814e-44f8-885e-9db3b502a960.png)